### DISCUSSION TITLE
Demo for a beginner NooB?

### DISCUSSION THREAD

**Discussion by rayvermey** [BurningTreeC/tiddlywiki-muuri#4] [Q&A]
Hi

i installed the plugin in my TW5 and see the extra new options..
But can someone please show or tell me how i can create a grid view??
I am rather a beginner so a step by step approach is very much appreciated.

Thanks a lot!

Ray

**Comment by BurningTreeC** [comment]
Hello @rayvermey,

The simplest option to do this is to select the `muuri` storyview under Tools -> storyview in the right SideBar. If you do so, the StoryRiver becomes a grid view.

If you want to create your own grid view have a look at [the documentation](https://burningtreec.github.io/tiddlywiki-muuri/#Configuration%20Options)

A grid view starts with a `list` widget that uses the `muuri` storyview and defines a configuration namespace, like for example:

```
<$list filter="[list[$:/myGridViewList]]" storyview="muuri" storyViewConfig="$:/config/myGridView/"/>
```

If you have more questions, please let me know

best wishes,
BTC

**Reply by rayvermey** [reply]
Hi, thanks, I made tiddler containing: <$list filter="[list[$:/myGridViewList]]" storyview="muuri" storyViewConfig="$:/config/myGridView/"/>
Choose the Muuri view.
Then i still have a tiddler accross the whole width of the normal story river.
So how can i make "tiles" of the tiddlers?
I would like to mimic google keep so i can ditch that app :-)
Please forgive my ignorance.

Ray

**Reply by BurningTreeC** [reply]
Hi @rayvermey ,

I believe in your case you don't need to create that tiddler, so you can delete it.

To have a tiled view, simply click the button shown in the screenshot below:

![Bildschirmfoto vom 2021-03-02 16-41-50](https://user-images.githubusercontent.com/9407182/109673704-93be1f00-7b76-11eb-832a-1c2783bb4d2b.png)

**Reply by BurningTreeC** [reply]
If you don't see that button, you can enable it under "Tools" in the right SideBar

**Comment by rayvermey** [comment]
Yes i have that button, i click it, it shows 2, i click again , it is 3
column... but.... the tiddlers are still across the whole screen...

HOW on earth can i achieve a view like i see here:
https://burningtreec.github.io/tiddlywiki-muuri/

I still can not get this working... aaaarggghhhhh :-)

Ray

Op di 2 mrt. 2021 om 16:53 schreef Simon Huber <notifications@github.com>:

**Reply by BurningTreeC** [reply]
@rayvermey,

please check again if you have your storyview set to `muuri` under SideBar -> Tools -> storyview. If not, changing it to `muuri` should do the trick.

**Comment by rayvermey** [comment]
Hmmmm it looks like it is a plugin question....
I set up a new empty tw5 then the muuri plugin works as expected...
In my own tw5 it does not work...
I will try to keep deleting plugins until muuri works then...

Ray

Op di 2 mrt. 2021 om 18:24 schreef Ray Vermey <rayvermey@gmail.com>:

**Comment by rayvermey** [comment]
Strange...

almost all plugins deleted.... still problem of changing column does not
change display of tiddlers....


Op di 2 mrt. 2021 om 18:55 schreef Ray Vermey <rayvermey@gmail.com>:

**Comment by rayvermey** [comment]
Hi Simon, is it possible to have a shared screen session with you?
For instance i can use https://meet.jit.si/  for that.
Maybe in 10 minutes you can guide me here?
I think that there is some setting in my TW that blocks the tiling layout
of Muuri...
Hope you can make some time free ;-)

Tx

Ray

Op di 2 mrt. 2021 om 19:04 schreef Ray Vermey <rayvermey@gmail.com>:

**Reply by BurningTreeC** [reply]
Hello @rayvermey , yes we could do a meeting over Jitsi Meet, sure.

**Comment by rayvermey** [comment]
Great... what time??
I am available now :-)

Ray

Op wo 3 mrt. 2021 om 16:24 schreef Simon Huber <notifications@github.com>:

**Reply by BurningTreeC** [reply]
Let's do it now

**Comment by rayvermey** [comment]
https://meet.jit.si/TiddlyWiki

Ray

Op wo 3 mrt. 2021 om 16:31 schreef Ray Vermey <rayvermey@gmail.com>:
